constants.h

```diff
@@ -2,13 +2,17 @@
 #define __CONSTANTS_H__
 
 /* Temperature threshold that has to be exceed long enough to start defrosting. */
-const float TEMPERATURE_DELTA_TO_DEFROST = 4.0;
+const float TEMPERATURE_DELTA_TO_DEFROST = 5.0;
 
 /* Temperature threshold that triggers defrosting even though
  * MIN_HEATING_TIME would not be passed. */
 /* NOT USED */
 /* const float TEMPERATURE_DELTA_TO_FORCE_DEFROST = 10.0; */
 
+const float OUTDOOR_TEMPERATURE_TO_ENTER_OFF_STATE = 3.0;
+
+const float OUTDOOR_TEMPERATURE_TO_EXIT_OFF_STATE = 2.0;
+
```

Comment:
I'm a bit thinking that is just one C enough difference here and is 3 degrees already too warm..
But let's go with these and tune based on experiences...